Ajout des snowball

Thomysr opened the pull request:
Hello,

Voici la pull request pour l'issue : [[SnowBall] Allows you to pick up snowballs](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/issues/46)

Je suis dispo si besoin de modif :) 

Bonne journée / soirée

Thomysr committed:
snowball

joelwurtz commented:
Thanks a lot for this PR, even if snow is no more present we still want to merge this for the next time we have snow, can you rebase it ?

Thomysr commented:
Hello, désolé du délais, le rebase est fait.

Rigonkmalk committed:
feat(feat/lsmc_v2): wokirng on it

Kedrihan commented:
Hello, je me suis permis de rebase cette feature qui avait été entamée, comme l'hiver approche :)

J'ai juste adapté le menu comme depuis il est dans le soz-core, mais je n'ai rien touché d'autre.

Rigonkmalk committed:
feat(lsmc): add doors tuning

joelwurtz committed:
chore(repository): add a way to listen on repository change

joelwurtz committed:
feat(housing): init migrate to core

Laikker committed:
Migrate utils to core

Laikker committed:
November 2023 crimi update

AurelienFT committed:
Fix typo

NariieL committed:
Merge pull request #5410 from SOZ-Faut-etre-Sub/keychain-multiple-actions

[FEATURE] Le trousseau peut maintenant donner toutes les clés par type

NariieL committed:
Merge pull request #5409 from SOZ-Faut-etre-Sub/fix/uhu-tp

Fix UHU tp target destination

Rigonkmalk committed:
feat(lantern): delete old ymap lantern and add lantern_custom on fxmanifest

joelwurtz committed:
feat(housing): migrate housing to core

Moustash committed:
[UPDATE] Doubler prix de revente DMC

Resell price

joelwurtz committed:
feat(housing): fix repository listener, allow apartment zone to be null

Laikker committed:
Remove duplicate BankModule init

Moustash committed:
[UDPDATE] Double food capacity and refill

NariieL committed:
Merge pull request #5428 from SOZ-Faut-etre-Sub/fix/duplicate-server-BankModule-init

Remove duplicate BankModule init

NariieL committed:
Merge pull request #5426 from SOZ-Faut-etre-Sub/5425-double-dmc-resell

[UPDATE] Doubler prix de revente DMC

Laikker committed:
Fix handcuff.zip

Aurukh committed:
Delete native props

joelwurtz committed:
feat(housing): fix blip system, better notification and some housing fixes

Moustash committed:
[FIX] Inventory KG display

lint

joelwurtz committed:
feat(core): use automatic patch system to sync data

AurelienFT committed:
Fix phone cant be used after zip/cuff

AurelienFT committed:
Fix objects can be remove by all jobs

AurelienFT committed:
Fix unexpected logs.

AurelienFT committed:
Readd the two ped for amory

AurelienFT committed:
Update check vip clothes

Laikker committed:
Stall and blur when veh is upside down

Laikker committed:
Fix remove Unemployed salary

joelwurtz committed:
feat(housing): add temporary access

joelwurtz committed:
Merge pull request #5439 from SOZ-Faut-etre-Sub/fix/phone_after_zip

Fix first day after lua -> TS police

joelwurtz committed:
Merge pull request #5438 from SOZ-Faut-etre-Sub/fix-inventory-kg-display

[FIX] Inventory KG display

joelwurtz committed:
fix(jobs): fix multiple bells for same job, fix grade permission for recruite / promote / fire

joelwurtz committed:
Merge pull request #5445 from SOZ-Faut-etre-Sub/fix/jobs

fix(jobs): fix multiple bells for same job, fix grade permission for recruit / promote / fire

joelwurtz committed:
feat(housing): use classic teleport for admin teleport apartment

joelwurtz committed:
feat(housing): add senate tables and relations

joelwurtz committed:
feat(housing): add temporary access for senate apartment

Rigonkmalk committed:
fix(lsmc): fixing interior last return and add LSMC crew painting

Laikker committed:
Update zones for LSMC v2

NariieL committed:
Merge pull request #5451 from SOZ-Faut-etre-Sub/feat/lmsc-v2-zone

Update zones for LSMC v2

NariieL committed:
Merge pull request #5427 from SOZ-Faut-etre-Sub/fix/5398-november-crimi

november crimi update

Laikker committed:
Fix LSMC v2 zones

Rigonkmalk committed:
chore(lsmc): delete audio glitch

joelwurtz committed:
feat(housing): add senate management, support temporary access to enter apartment

Laikker committed:
Fix crimi veh siphon

NariieL committed:
Merge pull request #5456 from SOZ-Faut-etre-Sub/fix/5454-criminel-megaticket

Fix crimi veh siphon

Laikker committed:
Fix some ZeraGirl inversion + small racket issues

joelwurtz committed:
feat(housing): use apartment id for house garage

joelwurtz committed:
fix default grade when recruiting

AurelienFT committed:
Fix binoculars can't ptt

AurelienFT committed:
Change banner on promote

AurelienFT committed:
Do not display grades that are not possible to promote to

joelwurtz committed:
feat(housing): allow to set senate for housing and player in game

Laikker committed:
Move FDO equipement only to decal to keep undershirt free for players

joelwurtz committed:
fix(housing): better blips, fix inside with senate, fix admin house

AurelienFT committed:
Fix banner jobs.

Laikker committed:
Store stonk bag collect history with citizenId and not source

joelwurtz committed:
Merge pull request #5475 from SOZ-Faut-etre-Sub/fix/promote_too_high

Do not display grades that are not possible to promote to

joelwurtz committed:
Merge pull request #5474 from SOZ-Faut-etre-Sub/fix/banner_job

Change banner on promote menu

joelwurtz committed:
Merge pull request #5473 from SOZ-Faut-etre-Sub/fix/ptt_binoculars

Fix binoculars can't PTT

joelwurtz committed:
Merge pull request #5479 from SOZ-Faut-etre-Sub/fix/5422-stonk-jewel-bag-collect

Store stonk bag collect history with citizenId and not source

joelwurtz committed:
fix(housing): fix adding roomate

Laikker committed:
Remove money after stock check for clohtes

joelwurtz committed:
feat(housing): add senate party member to player

Laikker committed:
Resize ZeraGirl illustration to 500x500 ti gain size

joelwurtz committed:
fix(housing): fix access for senate party id

joelwurtz committed:
feat(housing): allow to set apartment tier from admin

joelwurtz committed:
Merge pull request #5482 from SOZ-Faut-etre-Sub/fix/zeragirl-resize

Resize ZeraGirl illustration to 500x500 ti gain size

joelwurtz committed:
Merge pull request #5352 from SOZ-Faut-etre-Sub/fix/appart_position_south

fix(appart): Move mlo appart

joelwurtz committed:
Merge pull request #5351 from SOZ-Faut-etre-Sub/feat/appart_forum

Feat(map): new apparts on Forum Drive and Peaceful Street

joelwurtz committed:
Merge pull request #5145 from SOZ-Faut-etre-Sub/fix/pixelized_txd

feat(clothes): optimized mipmas; color

joelwurtz committed:
feat(housing): be nice with culling

joelwurtz committed:
feat(housing): close menu when too far

Laikker committed:
Remove veh from currentFilling list in case of error

joelwurtz committed:
feat(housing): fix give keys to player

Laikker committed:
Fix police remove license point

joelwurtz committed:
feat(housing): fix removing house from senat party

joelwurtz committed:
Merge pull request #5487 from SOZ-Faut-etre-Sub/fix/remove-license-point

Fix police remove license point

Laikker committed:
Job boss see all grades in promotion menu

Laikker committed:
Fix store veh in private/job garage

Laikker committed:
lint

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Laikker committed:
Fix Bank module included twice

Laikker committed:
Fix only one place in trailer garage

Laikker committed:
Fix garage

joelwurtz committed:
Merge pull request #5498 from SOZ-Faut-etre-Sub/fix/tailer-garage-max-place

Fix only one place in trailer garage

Laikker committed:
Fix spawn of drug props in apartment

Laikker committed:
Update housing.provider.ts

Laikker committed:
Housing check entry zone

NariieL committed:
Merge pull request #5500 from SOZ-Faut-etre-Sub/fix/drugs-props-apartment

Fix spawn of drug props in apartment

Laikker committed:
Set core blips default range to 0.8 as in QB

joelwurtz committed:
fix(party): fix party member not correctly set, reduce patch tick to avoid too much cpu

joelwurtz committed:
fix(housing): do not allow to bell if access

joelwurtz committed:
fix blips for upw

joelwurtz committed:
add animation for house

joelwurtz committed:
Merge pull request #5525 from SOZ-Faut-etre-Sub/fix/party

Fix housing & stuff

Rigonkmalk committed:
Revert "feat(clothes): optimized mipmas; color"

This reverts commit 93f3b582c3ed2dd2c78198f8dbba6a2c9d13b032.

Laikker committed:
Do flag veh as missing when in destroyed state

Laikker committed:
Prevent reactivate inventory focus after asking input in case of external close

Laikker committed:
Better display of remaning time before next car purchase

Laikker committed:
Fix marina boat spawn location for long boats

Laikker committed:
Fetch spike list on startup

Laikker committed:
Fix Zkea upgrade by passing property and apartement in correct order for housingRepository.getApartment

NariieL committed:
Merge pull request #5531 from SOZ-Faut-etre-Sub/fix/5503-load-spike-startup

Fetch spike list on startup

NariieL committed:
Merge pull request #5530 from SOZ-Faut-etre-Sub/fix/5505-marina-boat-spawn-location

Fix marina boat spawn location for long boats

NariieL committed:
Merge pull request #5529 from SOZ-Faut-etre-Sub/fix/5507-display-time-before-dealership-buy

Better display of remaning time before next car purchase

NariieL committed:
Merge pull request #5528 from SOZ-Faut-etre-Sub/fix/5098-focus-external-close

Prevent reactivate inventory focus after asking input in case of external close

NariieL committed:
Merge pull request #5527 from SOZ-Faut-etre-Sub/fix/5333-missing-state

Do flag veh as missing when in destroyed state

Laikker committed:
Fix Zkea add parking place

Aurukh committed:
Delete aircon prop

joelwurtz committed:
feat(housing): fix bell, catch error on patch

joelwurtz committed:
Merge pull request #5537 from SOZ-Faut-etre-Sub/fix/5536-buy-parking-place-trailer

Fix Zkea add parking place

joelwurtz committed:
Merge pull request #5540 from SOZ-Faut-etre-Sub/fix/bell-patch

feat(housing): fix bell, catch error on patch

Laikker committed:
Add option to show all lantern on maps in admin menu

AurelienFT committed:
Add try catch to avoid soft lock

Moustash committed:
[DEV] MTP - Tenues / Puits

Aurukh committed:
fix(hammerplace): rebuild hammer place

joelwurtz committed:
feat(stream): readd cinema

Laikker committed:
Fix btarget player for job without bosszone

joelwurtz committed:
Merge pull request #5556 from SOZ-Faut-etre-Sub/fix/fib-btarget-player

Fix btarget player for job without bosszone

joelwurtz committed:
Merge pull request #5553 from SOZ-Faut-etre-Sub/fix/hammer_place

fix(hammerplace): fix crash and rebuild hammer place

joelwurtz committed:
Merge pull request #5547 from SOZ-Faut-etre-Sub/feat/hub-entry-all-blip

Add option to show all lantern on maps in admin menu

Laikker committed:
Add paparazzi animation

Laikker committed:
Remove Naked Config when changing clothe in house closet

Laikker committed:
Fix marked money checker

NariieL committed:
Merge pull request #5559 from SOZ-Faut-etre-Sub/fix/remove-naked-closet

Remove Naked Config when changing clothe in house closet

NariieL committed:
Merge pull request #5558 from SOZ-Faut-etre-Sub/feat/paparazzi-animation

Add paparazzi animation

AurelienFT committed:
Fix try catch on rope service

BernardJeremy committed:
fix(pdm): fix pdm display when veh stock is < 0

joelwurtz committed:
fix(model): correctly unload model when it's not needed anymore

NariieL committed:
Merge pull request #5568 from SOZ-Faut-etre-Sub/fix/unload-model

fix(model): correctly unload model when it's not needed anymore

NariieL committed:
Merge pull request #5567 from SOZ-Faut-etre-Sub/fix/display_no_stock_veh_concess

[PDM] Correction du display du label des véhicules ayant un stock négatif

NariieL committed:
Merge pull request #5564 from SOZ-Faut-etre-Sub/fix/marked-money-checker

Fix marked money checker

NariieL committed:
Merge pull request #5548 from SOZ-Faut-etre-Sub/fix/add_try_catch_rope

Add try catch in rope creation to avoid being locked

Laikker committed:
Temporary remove delay between vehicle buy

Laikker committed:
Revert some logic in Dealership show veh to fix veh not cleanup

Rigonkmalk committed:
fix(speedbump): fixed height of somes speedbump

Rigonkmalk committed:
fix(cabin): moved cabin for chantier in disabled

Rigonkmalk committed:
fix(globalmap): add static flags

Rigonkmalk committed:
chore(mapping): moved barrier

joelwurtz committed:
fix(dealership): remove blocking vehicle if not network in dealership

joelwurtz committed:
Merge pull request #5583 from SOZ-Faut-etre-Sub/fix/5272_globalmap

fix(globalmap): add static flags

joelwurtz committed:
Merge pull request #5582 from SOZ-Faut-etre-Sub/fix/0_0_0_col

Suppression du collision en 0 0 0

joelwurtz committed:
Merge pull request #5581 from SOZ-Faut-etre-Sub/fix/speedbump

fix(speedbump): fixed height of somes speedbump

joelwurtz committed:
Merge pull request #5579 from SOZ-Faut-etre-Sub/fix/small-revert-dealership-show-veh

Revert some logic in Dealership show veh to fix veh not cleanup

joelwurtz committed:
fix(dealership): unload model if not needed

joelwurtz committed:
Merge pull request #5493 from SOZ-Faut-etre-Sub/feat/migrate-utils-to-core

Migrate utils to core

Rigonkmalk committed:
feat(mtp): working on interior and exterior

joelwurtz committed:
fix(blip): fix heading for blips

Laikker committed:
Increase max bank refill per day

Laikker committed:
Fix noclip deleting non mission entity

NariieL committed:
Merge pull request #5587 from SOZ-Faut-etre-Sub/fix/bank-refill-limit

Increase max bank refill per day

NariieL committed:
Merge pull request #5585 from SOZ-Faut-etre-Sub/fix/blip-heading

fix(blip): fix heading for blips

Laikker committed:
Fix buying underderwear

Laikker committed:
Fix house clothset typo

joelwurtz committed:
feat(vehicle): disallow upgrading vehicle if bad condition or dirty

joelwurtz committed:
Merge pull request #5597 from SOZ-Faut-etre-Sub/fix/5594-house-clothset-typo

Fix house clothset typo

joelwurtz committed:
Merge pull request #5601 from SOZ-Faut-etre-Sub/feat/vehicle-no-custom-bad-condition

feat(vehicle): disallow upgrading vehicle if bad condition or dirty

Rigonkmalk committed:
feat(lsmc): add complete audio occlusion

Moustash committed:
Items and Tug

joelwurtz committed:
feat(luxury): do not count vehicle in luxury into stock

joelwurtz committed:
Merge pull request #5604 from SOZ-Faut-etre-Sub/feat/luxury-stock

feat(luxury): do not count vehicle in luxury into stock

Laikker committed:
Allow staff to operate stream screen

AurelienFT committed:
Fix moneycheck id server

joelwurtz committed:
feat(custom): be nicer when checking dirt level / windows broken

AurelienFT committed:
Fix money checker server

Rigonkmalk committed:
Revert "feat(lsmc): add complete audio occlusion"

This reverts commit 94363ac68734c0917c37437715bee1536dbe63b9.

NariieL committed:
Merge pull request #5622 from SOZ-Faut-etre-Sub/feat/lsmc_audio_occlusion

Revert "feat(lsmc): add complete audio occlusion"

NariieL committed:
Merge pull request #5620 from SOZ-Faut-etre-Sub/feat/custom-bad-condition

feat(custom): be nicer when checking dirt level / windows broken

AurelienFT committed:
Fix notif

NariieL committed:
Merge pull request #5621 from SOZ-Faut-etre-Sub/fix/money-checker

Fix/money checker

Laikker committed:
Rework vandalism props

Moustash committed:
Items and Images

joelwurtz committed:
fix(housing): fix resell price, fix clean house

joelwurtz committed:
Merge pull request #5626 from SOZ-Faut-etre-Sub/fishing-dlc-2-tug-delivery

[FISHING] Adding new fishes and TUG when 100%

joelwurtz committed:
Merge pull request #5629 from SOZ-Faut-etre-Sub/fix/housing-resell

fix(housing): fix resell price, fix clean house

NariieL committed:
Merge pull request #5631 from SOZ-Faut-etre-Sub/fix/4927-vandalisme-kérosène

Rework vandalism props

Laikker committed:
Fix cinematic view

Rigonkmalk committed:
mtp: add new wall

Rigonkmalk committed:
mtp: mirror logo

Moustash committed:
[FIX] Sozedex fish infos

joelwurtz committed:
feat(housing): publish event on house upgrade

joelwurtz committed:
Merge pull request #5640 from SOZ-Faut-etre-Sub/fix-sozedex-fish-weather

[FIX] Sozedex fish infos

Moustash committed:
Augmentation délai production puits

joelwurtz committed:
Merge pull request #5549 from SOZ-Faut-etre-Sub/5516-MTP-folder

[Dossier MTP]

Moustash committed:
FIX - fish id 190 name

Fix

Rigonkmalk committed:
mtp: multiple fix sur retour

NariieL committed:
Merge pull request #5648 from SOZ-Faut-etre-Sub/5647-fix-fish-illustrator

FIX - fish id 190 name

Rigonkmalk committed:
mtp: fixed ground and holes

Laikker committed:
Add monitoring event at start of store vehicule

joelwurtz committed:
feat(bennys): lower constraint on bad condition for custom

Moustash committed:
[AJOUT] Hameçon détritus

Moustash committed:
[FIX] ELectric vehicle

Moustash committed:
Typo

joelwurtz committed:
Merge pull request #5652 from SOZ-Faut-etre-Sub/fix/mtp_last_return

mtp: Mauvaise exportation du sol

joelwurtz committed:
Merge pull request #5654 from SOZ-Faut-etre-Sub/feat/event-start-store-veh

Add monitoring event at start of store vehicule

joelwurtz committed:
Merge pull request #5656 from SOZ-Faut-etre-Sub/fix/bad-condition

feat(bennys): lower constraint on bad condition for custom

joelwurtz committed:
Merge pull request #5659 from SOZ-Faut-etre-Sub/5658-fishing

[AJOUT] Hameçon détritus

joelwurtz committed:
Merge pull request #5661 from SOZ-Faut-etre-Sub/5660-electric-vehicle

[FIX] ELectric vehicle

Laikker committed:
Always display Cayo may with ExtraMapTiles lib

Laikker committed:
14 days expiration on meal box

Rigonkmalk committed:
clothes: add new heels

Aurukh committed:
fix flying tree

Aurukh committed:
fix zkea lod

NariieL committed:
Merge pull request #5663 from SOZ-Faut-etre-Sub/fix/cayo-map

Always display Cayo may with ExtraMapTiles lib

MimiBloops committed:
Fixing Gunsmith error message

MimiBloops committed:
Fixing giving not whole quantity

MimiBloops committed:
Changing info traffic into info trafic

Laikker committed:
Fix hammer transder for staff/admin

Laikker committed:
Add monitoring event for baun harvest

Rigonkmalk committed:
mtp: fixed entities floating in the air

NariieL committed:
Merge pull request #5676 from SOZ-Faut-etre-Sub/fix/gunsmith-error-message

Fixing Gunsmith error message

NariieL committed:
Merge pull request #5682 from SOZ-Faut-etre-Sub/fix/admin-transfer-hammer

Fix hammer transder for staff/admin

NariieL committed:
Merge pull request #5683 from SOZ-Faut-etre-Sub/feat/baun-harvest-monitoring

Add monitoring event for baun harvest

NariieL committed:
Merge pull request #5678 from SOZ-Faut-etre-Sub/fix/give-not-whole-quantity

Fixing giving not whole quantity

NariieL committed:
Merge pull request #5679 from SOZ-Faut-etre-Sub/fix/tn-info-trafic

Changing info traffic into info trafic

Laikker committed:
December balancing

Laikker committed:
Fix vandalism disable radar

NariieL committed:
Merge pull request #5686 from SOZ-Faut-etre-Sub/feat/5653-decembre-update

December balancing

Laikker committed:
Set player Btarget police range to default

Laikker committed:
Fix zeed joint fx

MimiBloops committed:
Fixing giving item

NariieL committed:
Merge pull request #5696 from SOZ-Faut-etre-Sub/fix/give-item-inventory

Fixing giving item

NariieL committed:
Merge pull request #5694 from SOZ-Faut-etre-Sub/fix/error-message-zeed-joint

Fix zeed joint fx

Rigonkmalk committed:
fix(lowr): fixed pants texture misplaced

Laikker committed:
Fix sync of animal no spawn after hunting

Laikker committed:
Revert to gunshot alert per player

joelwurtz committed:
fix lua

joelwurtz committed:
Merge pull request #5699 from SOZ-Faut-etre-Sub/fix/lowr_pants

fix(lowr): fixed pants texture misplaced

joelwurtz committed:
Merge pull request #5700 from SOZ-Faut-etre-Sub/fix/hunt-no-spawn-sync

Fix sync of animal no spawn after hunting

joelwurtz committed:
Merge pull request #5701 from SOZ-Faut-etre-Sub/feat/shooting-alert-delay-per-player

Revert to gunshot alert per player

Moustash committed:
[FIX] Tenues MTP

Laikker committed:
Destroy gaz station on explosion

Laikker committed:
Fix check plate for streiter

NariieL committed:
Merge pull request #5710 from SOZ-Faut-etre-Sub/fix/5669-streiter-plate-check

Fix check plate for streiter

Laikker committed:
Prevent use of vehicle menu when in coma

NariieL committed:
Merge pull request #5711 from SOZ-Faut-etre-Sub/fix/5644-coma-open-veh-menu

Prevent use of vehicle menu when in coma

Laikker committed:
Better kill info retrieval on car kill

Laikker committed:
Send police alert for drug sell at begin/middle of animation

Laikker committed:
Fix monitoring import for hunting no spawn

Laikker committed:
Regularly freeze payer ped when coma to try to avoid desync

Laikker committed:
Fix docks paking location for big ships

Laikker committed:
Display policeMessageCount in society message list for fdo

NariieL committed:
Merge pull request #5720 from SOZ-Faut-etre-Sub/fix/docks-boat-location

Fix docks paking location for big ships

NariieL committed:
Merge pull request #5718 from SOZ-Faut-etre-Sub/fix/4668-coma-player-not-visible

Regularly freeze payer ped when coma to try to avoid desync

NariieL committed:
Merge pull request #5716 from SOZ-Faut-etre-Sub/fix/hunt-no-spawn-monitoring

Fix monitoring import for hunting no spawn

NariieL committed:
Merge pull request #5715 from SOZ-Faut-etre-Sub/feat/5707-drug-police-alert

Send police alert for drug sell at begin/middle of animation

NariieL committed:
Merge pull request #5713 from SOZ-Faut-etre-Sub/fix/car-kill-info

Better kill info retrieval on car kill

Rigonkmalk committed:
feat(housing): add north low and mid; trailer

feat(housing): fixing v_studio_lo portals

feat(housing): add new housing from Oneiluj new housing batch

Rigonkmalk committed:
fix(housing): fixed ytd for mid shell

Laikker commented:
Intégré
